ldawithgridsearch.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pandas as pd
+import re, nltk, spacy, gensim
+# SKLearn
+from sklearn.decomposition import LatentDirichletAllocation, TruncatedSVD
+from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer
+from sklearn.model_selection import GridSearchCV
+
+dtm = pd.read_pickle("dtm.pkl")
+
+
+# Build LDA Model
+def lda_grid_search(n1, n2, n3, d1, d2, d3):
```

Comment:
This function signature would spark the following point. I understand the parameters of the function are what the LDA model requires. However, this would not be evidently clear to another developer if they are not familiar with the LDA algo. Hence I would suggest one of two approaches when writing functions: i) use human understandable names, or ii) in cases like these where one-letter variables make sense or have a meaning, docstrings can help.